modules/rosa-deleting-cluster.adoc

```diff
@@ -17,11 +17,11 @@ ifdef::sts[]
 endif::sts[]
 
 ifndef::sts[]
-You can delete a {product-title} cluster using the ROSA CLI (`rosa`).
+You can delete a {product-title} cluster using the {product-title} (ROSA) CLI (`rosa`).
```

Comment:
First mention should be "{product-title} (ROSA)", then you can just say "ROSA CLI (`rosa`)" at the end, here and in the next paragraph